### ISSUE TITLE
Freeze agenda for high risk call agenda at midday the day before the call 

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/whatdotheyknow-theme#1557]
Currently cases which have emerged just hours/days before the call get discussed. 

We could move to an approach of fixing the agenda at say midday the day before the call, and not discussing cases which emerge after that point.  This would be in-line with the latest practice for the Community and Activism Call. 

Developments on cases on the agenda could still be presented on the call. 

A disadvantage is the removal of the opportunity to discuss the newly emergent cases at an early stage.

**Comment by HelenWDTK** [comment]
I think this is good idea. I think we work in a flexible enough way that should something really serious and urgent happen in that period, we could adapt the running order of the meeting to take it into account., and we do have AOB for things like that. 

If there are no objections, I will try this out for the next call as a trial to see how it goes. 

I think there may be a benefit to updating the tracked metrics on the morning of the call, but there is no reason why a snapshot couldn't be taken at an earlier point of time instead.

**Comment by RichardTaylor** [comment]
We should consider how AOB is to be used. 

Is it for items which don't fit in other categories, but items have to be on the agenda before it is frozen?

Is it for late items (a practice I've seen elsewhere is to ask for AOB items at the start of the meeting and add them, if agreed, at that point).

**Comment by RichardTaylor** [comment]
We had an experience of the new approach today. 

>We should consider how AOB is to be used.

Today we had no AOB items on the agenda, and no items were added before we reached the AOB section. 

When we reached the AOB section we briefly discussed cases/issues which had arisen since the agenda was frozen, but without the benefit of any summaries/links/proposals on them. Having this discussion rather undermines the point of freezing the agenda. 

Perhaps we should mention notable aspects of issues which have arisen since the agenda was frozen, but note we're doing so in a relatively unprepared fashion - sharing first thoughts - which might be wrong. 

**Related issue (but veering off topic):**

The method of getting policy items on the agenda isn't clear, the process we have is solely focused on cases https://wdtkwiki.mysociety.org/wiki/Review_process 

I'm wondering if I need to start a review thread on a policy matter to get it on the agenda, or if just having ticket ad adding it suffices.
